resources/docs/usage.rst

```diff
@@ -28,7 +28,7 @@ You can use the `Query Builder`_:
 
 .. caution::
 
-    Don't forget to `enable the DB aliases`_ if you want to use the ``DB``
+    Don't forget to [enable the DB aliases](configuration.rst) if you want to use the ``DB``
```

Comment:
ah indeed, will try. This is also certainly why https://github.com/wouterj/WouterJEloquentBundle/pull/25#discussion_r81459631 fails